### ISSUE TITLE
Frigate-hass-card not available to be added to dashboard

### ISSUE THREAD

**Issue by Klaus062** [dermotduffy/frigate-hass-card#956]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:** v4.0.0

**Last working release (if known):** n/a

**Browser and Operating System:** Firefox 102.8.0esr on Kubuntu 22.04

**Description of problem:**
Running HAOS in a Virtualbox with Kubuntu 22.04 being the host and installed frigate with HACS, integration v3.0.1
Downloaded frigate-hass-card with HACS (frontend) v4.0.0 
frigate-hass-card is not available as card to be added to the dashboard. It is just not available. Several HA restart, reboots of VM and deleting browser cache. Even rebooting host computer several times - no way to get the card. Tried to add 
```yaml
lovelace:
  resources:
    - url: /hacsfiles/frigate-hass-card/frigate-hass-card.js
      type: module
```
to the configuration.yaml just in case, with several restarts and reboots, but still no change.  

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**

**Comment by dermotduffy** [comment]
Thanks for the report @Klaus062 .

Can you visit `[YOUR HA INSTANCE WITH PORT]/hacsfiles/frigate-hass-card/frigate-hass-card.js` and do you see Javascript returned? If not, the card has not been installed correctly. 

If yes, do you see anything in your Javascript console when you load the dashboard? (Press 'F12'). When you hit the blue 'Add Card' button on a dashboard, you do not see Frigate card in the resulting menu?

**Comment by Klaus062** [comment]
Thanks a lot for your feedback - that indeed was the problem. No Javascript feedback from [YOUR HA INSTANCE WITH PORT]/hacsfiles/frigate-hass-card/frigate-hass-card.js. 

First uninstall and re-install Frigate-hass-card did not help. I then went back to a HASS backup taken prior frigate installation and started from there again installing frigate - integration and Frigate-hass-card and now it works fine.

Seems that I messed up something during my first try.

Thanks a lot for your help
Klaus
